docs/content/manual/_incoming/1831_Node_disk_manager_should_prevent_too_many_concurrent_disk_formatting_occur_within_a_short_period.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Node disk manager should prevent too many concurrent disk formatting occur within a short period
+category: console
+tags: dashboard, p2, functional
+---
+Ref: https://github.com/harvester/harvester/issues/1831
+
+
+### Criteria
+- [x] exceed the maximum, there should have requeue devices which equals the exceeds
+- [x] hit the maximum, there should not have requeue devices
+- [x] less than maximum, there should not have requeue devices
+
+![image](https://user-images.githubusercontent.com/5169694/177324553-3b4800b2-9db9-45ec-a3cf-a630acb384cf.png)
+
+
+### Verify Steps:
+1. Install Harvester with any node having at least 6 additional disks
```

Comment:
Do the disks need to be of any set size? Like 1GB, 2GB, 2.5GB, etc.?  

Or any set type?
Like: virtio, sata, scsi, or usb?

Reply:
any types

Reply:
sounds good :smile: